tests/main/base-policy/task.yaml

```diff
@@ -0,0 +1,46 @@
+summary: Ensure the default policy is correct for base and core bases
+
+prepare: |
+    echo "Given a basic snap is installed"
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local test-snapd-sh-default-base
+    install_local test-snapd-sh-core18
+    install_local test-snapd-sh-other18
+
+execute: |
+    if [ "$(snap debug confinement)" = none ] ; then
```

Comment:
I was expecting this to say != `strict`, we don't have `none` confinement IIRC.

Reply:
Hmm, I grabbed this from other tests. Since the tests only need 'file' from apparmor, only partial apparmor was needed. I'll take another look.

Reply:
Done